File: output/kotlin/models/serializable_data_class/StackTraceElem.kt
package data_class

@Serializable
data class StackTraceElem(
    @SerialName("is_native_method") val isNativeMethod: Boolean,
    @SerialName("method_name") val methodName: String,
    @SerialName("file_name") val fileName: String,
    @SerialName("line_number") val lineNumber: Int,
    @SerialName("class_name") val className: String,
)

File: output/golang/models/struct/StackTraceElem.go
package main

type StackTraceElem struct {
	IsNativeMethod *bool `json:"is_native_method" bson:"is_native_method"`
	MethodName *string `json:"method_name" bson:"method_name"`
	FileName *string `json:"file_name" bson:"file_name"`
	LineNumber *int `json:"line_number" bson:"line_number"`
	ClassName *string `json:"class_name" bson:"class_name"`
}